runtime/src/accounts_db.rs

```diff
@@ -10933,4 +11168,23 @@ pub mod tests {
         let stores = vec![Arc::new(s1), Arc::new(s2)];
         assert!(AccountsDb::is_shrinking_productive(0, &stores));
     }
+
+    #[test]
+    fn test_is_candidate_for_shrink() {
+        solana_logger::setup();
+
+        let mut accounts = AccountsDb::new_single();
+        let dummy_path = Path::new("");
+        let dummy_size = 2 * PAGE_SIZE;
+        let entry = Arc::new(AccountStorageEntry::new(&dummy_path, 0, 1, dummy_size));
```

Comment:
For clarity, maybe an assert, or an explicit assignment to make clear what the default assumed `accounts.shrink_ratio` is there?

Reply:
Yes. Will do